app/queries/servers/post.ts

```diff
@@ -10,6 +10,33 @@ import type PostInChannelModel from '@typings/database/models/servers/posts_in_c
 
 const {SERVER: {POST, POSTS_IN_CHANNEL}} = MM_TABLES;
 
+export const prepareDeletePost = async (post: PostModel): Promise<Model[]> => {
+    const preparedModels: Model[] = [post.prepareDestroyPermanently()];
+    const relations: (Relation<Model> | Query<Model>)[] = [post.metadata, post.drafts, post.postsInThread];
```

Comment:
you could exclude metadata, is fine not to remove it as I'm removing the entire table in a later PR